templates/cli/lib/utils.js.twig

```diff
@@ -67,7 +67,13 @@ function showConsoleLink(serviceName, action, open, ...ids) {
 
     if (open) {
         const start = (process.platform == 'darwin' ? 'open' : process.platform == 'win32' ? 'start' : 'xdg-open');
-        cp.exec(`${start} ${url}`);
+
+        cp.exec(`${start} ${url}`, (err, stdout, stderr) => {
+            if (err !== null) {
+                console.log('\n');
+                error('Opening in default browser. ' + err)
+            }
+        });
```

Comment:
Do we need this --open option? I think it might cause more problems than we are trying to solve. Does `xdg-open` work on all linux distros? 

I think we can save this trouble altogether by removing the `open` param. We can just display the link with the --console flag and people can just click it.